drivers/health_pctp/app/views/health_pctp/careplans/edit.haml

```diff
@@ -1,3 +1,4 @@
+.pull-right= render 'report_downloads/report_download', export: @pdf_export, excel_export: nil, excel_download_path: nil
```

Comment:
I'm not sure where we've done this for the health side before, but I'm guessing this needs a different partial, or mechanism so the links point to the health database.  @awisema ?

Reply:
@meborn Did you find this? It looks to me like there is not a reusable partial for the health side, but maybe `app/views/warehouse_reports/health/agency_performance/_report_download.haml` is a start?